test/Issue.test.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-undef, no-unused-expressions */
+const { expect } = require('chai');
+const sinon = require('sinon');
+require('chai').use(require('sinon-chai'));
+const Issue = require('../src/models/Issue');
+
+describe('Issue model', () => {
+  it('its constructor accepts a passed in argument', () => {
```

Comment:
@michaelmurray6298 imagine reading this line starting with the word "it". With that in mind I would recommend `it('should have a constructor that accepts an arguement', () => {`